jmix-superset/superset-flowui/src/main/java/io/jmix/supersetflowui/SupersetFlowuiProperties.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.supersetflowui;
+
+import io.jmix.supersetflowui.component.SupersetDashboard;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.bind.DefaultValue;
+
+import java.time.Duration;
+
+@ConfigurationProperties("jmix.supersetflowui")
+public class SupersetFlowuiProperties {
+
+    /**
+     * Defines a timeout of background task that fetches guest token for {@link SupersetDashboard} component
```

Comment:
For configuration properties we usually use the following javadoc pattern: "Timeout for background task..." (starts with noun). See other configuration classes. The reason is that this text is displayed by IntelliJ Idea when it suggests property name while editing the application.properties file.

Reply:
Fixed